azure-kusto-ingest/azure/kusto/ingest/ingestion_properties.py

```diff
@@ -69,15 +71,75 @@ class ReportMethod(IntEnum):
     Queue = 0
 
 
+class TransformationMethod(Enum):
+    # Transformations to configure over json column mapping
```

Comment:
please document this with triple quotes """ like other places, and preferably link to docs